### PR TITLE
Create ui-c8y-1018-503-82-uploading-package-versions-via-ui-now-take-the-correct-limit-graft.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-58818](https://cumulocity.atlassian.net/browse/MTM-58818)
Original PR: [6193](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6193)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1018-503-82-uploading-package-versions-via-ui-now-take-the-correct-limit-graft.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Uploading package versions via UI now take the correct limit. [GRAFT][release/y2024] (#6193)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-82-uploading-package-versions-via-ui-now-take-the-correct-limit-graft.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Increased package version upload limit in user interface
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-82-uploading-package-versions-via-ui-now-take-the-correct-limit-graft.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58818
+version: 1018.503.82
+---
+Uploading package versions via UI now take the correct limit. [GRAFT][release/y2024] (#6193)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-82-uploading-package-versions-via-ui-now-take-the-correct-limit-graft.md:17]: The Claude AI suggested to use the following body:
```suggestion
Previously, the user interface incorrectly limited the number of package versions that could be uploaded in one request. This has now been fixed to match the intended higher limit. Users can now successfully upload a larger number of package versions in one request through the user interface, making the process more efficient and convenient.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1018-503-82-uploading-package-versions-via-ui-now-take-the-correct-limit-graft.md (hunk 1 above)
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1018-503-82-uploading-package-versions-via-ui-now-take-the-correct-limit-graft.md:3]:
```suggestion
title: The package version upload limit via the user interface has been increased.
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-82-uploading-package-versions-via-ui-now-take-the-correct-limit-graft.md (hunk 2 above)
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1018-503-82-uploading-package-versions-via-ui-now-take-the-correct-limit-graft.md:17]:
```suggestion
Previously an incorrect limit of 6 package versions could be uploaded via the user interface. It is now possible to upload a higher number of package versions. The limit is 20 by default. 
```

**Review by eniosultan — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1018-503-82-uploading-package-versions-via-ui-now-take-the-correct-limit-graft.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58818
+version: 1018.503.82
+---
+Previously an incorrect limit of 6 package versions could be uploaded via the user interface. This has now been fixed to match the intended higher limit. The limit is 20 by default.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1018-503-82-uploading-package-versions-via-ui-now-take-the-correct-limit-graft.md:17]:
```suggestion
The number of package versions that could be uploaded via the UI was incorrectly limited to 6. This has been fixed to match the default limit of 20 package versions. 
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-82-uploading-package-versions-via-ui-now-take-the-correct-limit-graft.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: The package version upload limit via the user interface has been increased.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1018-503-82-uploading-package-versions-via-ui-now-take-the-correct-limit-graft.md:3]:
```suggestion
title: Package version upload limit via UI has been increased
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-82-uploading-package-versions-via-ui-now-take-the-correct-limit-graft.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Package version upload limit via UI has been increased to match the default
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1018-503-82-uploading-package-versions-via-ui-now-take-the-correct-limit-graft.md:3]:
```suggestion
title: Package version upload limit via UI increased to match the default
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**